install_config/configuring_sdn.adoc

```diff
@@ -18,11 +18,31 @@ network_. Two xref:../architecture/additional_concepts/sdn.adoc#architecture-add
 are currently available (*ovs-subnet* and *ovs-multitenant*), which provide
 different methods for configuring the pod network.
 
+== Configuring the Pod Network with Ansible
+
 For initial xref:../install_config/install/advanced_install.adoc#install-config-install-advanced-install[advanced
 installations], the *ovs-subnet* plug-in is installed and configured by default,
 though it can be
-xref:../install_config/install/advanced_install.adoc#configuring-ansible[overridden
-during installation] using the `*os_sdn_network_plugin_name*` parameter.
+xref:../install_config/install/advanced_install.adoc#configuring-ansible[overridden during installation] using the
+xref:../install_config/install/advanced_install.html#configuring-cluster-variables[`*os_sdn_network_plugin_name*` parameter], which is configurable in the inventory file.
+
+.Example SDN Configuration with Ansible
+====
+
+----
+# Configure the multi-tenant SDN plugin (default is 'redhat/openshift-ovs-subnet')
+# os_sdn_network_plugin_name='redhat/openshift-ovs-multitenant'
+
+# Disable the OpenShift SDN plugin
+# openshift_use_openshift_sdn=False
+
+# Configure SDN cluster network CIDR block. This network block should
+# be a private block and should not conflict with existing network
+# blocks in your infrastructure that pods may require access to.
+# Can not be changed after deployment.
+#osm_cluster_network_cidr=10.1.0.0/16
+----
+====
 
 ifdef::openshift-enterprise[]
 For initial xref:../install_config/install/quick_install.adoc#install-config-install-quick-install[quick
```

Comment:
IMO, we have the statement: ["can be reconfigured post-installation"](https://github.com/openshift/openshift-docs/pull/2350/files#diff-2286e6c0ed74dfb182205830dcec2932R50) however we don't address this. The main point of the BZ we are discussing is to address these missing docs items, by providing examples and/or links to where users can use existing documentation to make these configuration points.